game/scripts/npc/abilities/bloodseeker_rupture.txt

```diff
@@ -29,11 +29,11 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "70 60 50 50 50"
+    "AbilityCooldown"                                     "70 60 50 45 40"
```

Comment:
Doesn't this sorta make his aghs nerfed? I suggest then reducing aghs cd to 30

Reply:
cd with aghs is 40. So its neither a buff or nerf. Also with aghs you get 2 charges.